### PR TITLE
chore(playwright): new reporter options for Allure TestOps compatibility

### PR DESCRIPTION
**Pull request by serembon**
### Context
When using the allure-playwright package, and uploading tests to Allure TestOps, the following inconveniences occur:

1. Reports contain Playwright`s API calls in substeps, and it hinders updating tests in TestOps.
api

2. By default, the suite label contains the path to the test file, and if the `allure.suite()` is declared, then the tests are duplicated, and `allure.suite()` is not parsed correctly.
suit

### Solution

Add options that can disable substeps with Playwright`s API calls  and not use the default value of the suite.

#### Checklist
- [x] [Sign Allure CLA][cla]
- [ ]  Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### Unit tests
If unit tests are needed, then I need help with the fixtures

### CODE REVIEW

File: packages/allure-playwright/README.md [hunk 1]
```diff
@@ -53,17 +53,31 @@ set ALLURE_RESULTS_DIR=my-allure-results
...
-or inside the config via:
+## Reporter options
+
+Some reporter settings can set by following options:
+
+| Option       | Description                                         | Default            |
+|--------------|-----------------------------------------------------|--------------------|
+| detail       | Hide `pw:api` and `hooks` steps in report           | `false`            |
+| outputFolder | Path to results folder                              | `./allure-results` |
+| suiteTitle   | Use test tittle instead of test file path in report | `false`            |
```
Comment (baev) [inline packages/allure-playwright/README.md:64]: typo

Comment (baev) [reply packages/allure-playwright/README.md:64]: BTW the description is unclear

**Review by baev — CHANGES_REQUESTED** [review]
please revert all the formatting changes

File: packages/allure-playwright/README.md (hunk 1 above)
Comment (serembon) [reply packages/allure-playwright/README.md]: Done

**Review by baev — APPROVED** [review]
Looks good!

**Comment by baev** [comment]
BTW I rather introduce bunch of new labels instead of changing the default one. Allure TestOps provides an options to change the grouping and use any other label (mapped to custom field). But it's much harder to change the behaviour for Allure Report users

**Pull request merged**